### PR TITLE
Navigation labeling

### LINKED ISSUE #28: Show navigation i settingview
I would like to have a titel or something to indicate the navigation in the settings view. It is not nessesary on the ohter screens since they show their navigation in the buttom navigation but when entering settings the buttom navigation is still marked on the view that you enterd settings from. 

An alternative could be to use something like how it looks when entering the DatePicker

### LINKED ISSUE #50: Show title of current view
It gets a bit confusing navigating, so we need to add some text in the top bar to signal where we are.

### PR DESCRIPTION
**Pull request by malin-mallan**
Added titles on every view

Comment out notification since unused.

### CODE REVIEW

File: src/language/en.tsx [hunk 1]
```diff
@@ -53,6 +56,28 @@ const en = {
     informationCardValuesscreen: 'Valuesscreen',
     informationCardCalenderscreen: 'Calenderscreen',
     informationCardBehaviouralactivation: 'Behavioural activation',
+
+    routeNames: {
+        Settings: 'Settings',
+        History: 'History',
+        Activities: 'Register activity',
+        ActivityRegistration: 'Register activity',
```
Comment (Dolvur) [inline src/language/en.tsx:64]: Should the frontpage of the Activity view have the title Activities or Register activity? Both seem fine but activities is more similar to the other frontpage titles.

File: src/language/en.tsx [hunk 2]
```diff
@@ -1,7 +1,10 @@
 const en = {
     languageName: 'English',
 
-    valuesHeaderEvaluation: 'Values',
+    navigationLabelActivites: 'Registation',    // Unused
+    navigationLabelCalendar: 'Calendar',        // Unused
+    navigationLabelValues: 'Values',            // Unused
+    navigationLabelInformation: 'Information',  // Unused
```
Comment (Dolvur) [inline src/language/en.tsx:7]: If these are unused, shouldn't they be removed?

File: src/language/sv.tsx [hunk 3]
```diff
@@ -53,6 +56,28 @@ const sv = {
     informationCardValuesscreen: 'Värderingsvy',
     informationCardCalenderscreen: 'Kalendervy',
     informationCardBehaviouralactivation: 'Beteendeaktivering',
+
+    routeNames: {
+        Settings: 'Inställningar',
+        History: 'Aktivitetshistorik',
+        Activities: 'Registrera aktivitet',
+        ActivityRegistration: 'Registrera aktivitet',
```
Comment (Dolvur) [inline src/language/sv.tsx:64]: Same thing as above.

**Review by Dolvur — APPROVED** [review]
Everything seems like it looks good for me. I got some errors the first time I tried it but haven't been able to reproduce it. The titles work well with both English and Swedish

**Review by LukasJonsson99 — APPROVED** [review]
Looks good!

File: src/language/en.tsx (hunk 2 above)
Comment (malin-mallan) [reply src/language/en.tsx:7]: We wasen't sure if we wanted to add them later so we keept them and if we want them, we only need to change a bool

File: src/language/en.tsx (hunk 1 above)
Comment (malin-mallan) [reply src/language/en.tsx:64]: I think register activity. The other havet more like a front page than activity. Better to tell the user what to do in this case.

**Pull request merged**